operatorconfig/driverconfig/powerflex/v2.13.0/controller.yaml

```diff
@@ -206,8 +216,8 @@ spec:
             - name: socket-dir
               mountPath: /var/run/csi
         - name: driver
-          image: dellemc/csi-vxflexos:v2.10.1
-          imagePullPolicy: IfNotPresent
+          image: quay.io/dell/container-storage-modules/csi-vxflexos:v2.13.0
```

Comment:
Why are we changing imagepull policy to "Always" ?
Slight concern:
Will this change break min samples tests? Image is not defined in those files, so the value in operatorconfig will be used instead. 
It might make sense to use "nightly" tag in these files for now and we change it back to 2.13 later. Or we could keep the min tests at 2.12 for now.

Reply:
I have updated the tag to nightly with imagepullpolicy as "Always", It is required for nightly